kpops/component_handlers/kafka_connect/connect_wrapper.py

```diff
@@ -128,7 +128,7 @@ def get_connector_config(
         connector_config = config.dict(exclude_none=True, exclude_unset=True)
         if (name := connector_config.get("name")) and name != connector_name:
             raise ValueError(
-                "Kafka Connect config should be the same as component name"
+                "Connect config should be the same as component name"
```

Comment:
```suggestion
                "Connector name should be the same as component name"
```
Pay attention when copying ;-)